.ci/dockerfiles/Dockerfile.rocm.base

```
@@ -0,0 +1,73 @@
+    fi
+
+# Install ROCm from the prerelease multi-arch repo (follows the "new" layout:
+# install path /opt/rocm/core-<MAJOR>.<MINOR>, amdrocm-runtime-dev meta-package).
+# /etc/rocm-build.env records the resolved layout for build-rocm.sh to read.
+RUN ROCM_MAJOR=$(echo "${ROCM_VERSION}" | cut -d. -f1); \
+    ROCM_MINOR=$(echo "${ROCM_VERSION}" | cut -d. -f2); \
+    ROCM_INSTALL_PATH="/opt/rocm/core-${ROCM_MAJOR}.${ROCM_MINOR}"; \
+    printf 'ROCM_MAJOR=%s\nROCM_MINOR=%s\nROCM_BUILD_TYPE=%s\nROCM_INSTALL_PATH=%s\n' \
+        "${ROCM_MAJOR}" "${ROCM_MINOR}" "new" "${ROCM_INSTALL_PATH}" \
+        > /etc/rocm-build.env
+RUN . /etc/rocm-build.env && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)